### PR TITLE
Added plottable.js@1.0.0

### PR DESCRIPTION
**Pull request by punya**

### CODE REVIEW

**Comment by punya** [comment]
@christinaforney, just so you know

**Comment by PeterDaveHello** [comment]
Can you also add git auto-update config for it? Thanks.

**Comment by punya** [comment]
@PeterDaveHello I added auto-update config, please let me know if anything else is necessary.

File: ajax/libs/plottable.js/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "description": "A library for creating charts out of D3",
+  "version": "1.0.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/palantir/plottable.git"
+  },
+  "author": "Palantir Technologies",
+  "autoupdate": {
+    "source": "git",
+    "target": "git@github.com:palantir/plottable.git",
```
Comment (PeterDaveHello) [inline ajax/libs/plottable.js/package.json]: Should use git protocol here, not ssh.

**Pull request merged**